app/assets/scripts/components/map.js

```diff
@@ -261,7 +261,13 @@ const Map = React.createClass({
 
   renderOverlayLegend: function (scale) {
     const isQuantile = scale.hasOwnProperty('invertExtent');
-    const iterable = (isQuantile ? scale.range() : scale.domain()).sort();
+    let iterable = (isQuantile ? scale.range() : scale.domain()).sort();
+    const category = this.props.overlay.category.toLowerCase();
+    const lookup = {'very low': 5, 'low': 4, 'medium': 3, 'high': 2, 'very high': 1, 5: 'very low', 4: 'low', 3: 'medium', 2: 'high', 1: 'very high'};
+    if (category === 'categorical') {
+      iterable = iterable.map((cat) => lookup[cat]).sort();
```

Comment:
If I'm understanding, this would just translate 'very low' to 5, and vice versa? 

So if the categories from the API were `[1, 4, 5]` you would get `['very high', 'low', 'very low']`, which would not sort in the way you want it to.